Assets/Scripts/Items/JumpTimeDecreaseItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class JumpTimeDecreaseItem : MonoBehaviour
+{
+    public float decreaseInSeconds;
+
+    void OnTriggerEnter2D(Collider2D col) {
+        if (col.tag == "Player") {
```

Comment:
Nit: `CompareTag(string)` is better than direct comparison performance-wise ([Source](https://github.com/JetBrains/resharper-unity/wiki/Use-CompareTag-instead-of-explicit-string-comparison))